src/mds/Server.h

```diff
@@ -347,6 +347,7 @@ class Server {
   double cap_revoke_eviction_timeout = 0;
   uint64_t max_snaps_per_dir = 100;
   unsigned delegate_inos_pct = 0;
+  int bal_fragment_size_max = 100000; 
```

Comment:
Since this is set in the constructor, I think this should remain uninitialized.

Reply:
Ack.